environment.yaml

```diff
@@ -1,4 +1,4 @@
-name: sd-ldm
```

Comment:
Why?

Reply:
This looks like it's from [PR 528](https://github.com/lstein/stable-diffusion/pull/528). It got pulled into my fork during a sync.

Reply:
Understood, I just spotted #528